index.html

```diff
@@ -183,9 +183,9 @@ <h1 class="ds-u-color--white">The Latest & Greatest from BCDA!</h1>
         <div class="ds-u-fill--gray-lightest ds-u-padding--3">
           <img hspace=0 vspace=0 src="/assets/img/homepage/logos/BB2.0_logo.svg">
           <div class="ds-u-padding-y--1">
-            Blue Button 2.0 API enables beneficiaries to connect their Medicare claims data to the applications, services, and research programs they trust.
+            The Blue Button 2.0 API enables beneficiaries to connect their Medicare claims data to the applications, services, and research programs they trust.
           </div>
-          <div class="ds-l-container ds-u-padding-top--2">
+          <div class="ds-l-container ds-u-padding-top--5">
             <div class="ds-l-row ds-u-justify-content--center">
               <a href="https://bluebutton.cms.gov/" target="_blank">
                 <button type="button" class="ds-c-button ds-c-button--primary ds-u-justify-content--center ds-l-col--12">
```

Comment:
@mtrang1263 - here, and elsewhere, it looks like we've created buttons by styling links as buttons. If so, this is a big miss from an accessibility standpoint. Can we discuss?

Reply:
Spoke to Elissa offline about this, since we're linking out to another page, ended up using the `<a>` tag and kept the button styling.